pkg/api/controllers/fileshare.go

```diff
@@ -50,6 +54,26 @@ func (f *FileSharePortal) CreateFileShare() {
 		log.Error(reason)
 		return
 	}
+
+	// Get profile
+	var prf *model.ProfileSpec
+	var err error
+	if fileshare.ProfileId == "" {
```

Comment:
Is profileId being passed? If yes, then why the Else condition

Reply:
If it is not passed it should use default profile that is the reason and there will be warning message log.Warning("Use default profile when user doesn't specify profile.")